docs/sources/reference/cli/environment-variables.md

```diff
@@ -52,28 +51,30 @@ You can use the `HTTPS_PROXY` environment variable to define the proxy server fo
 
 The `NO_PROXY` environment variable is used to define any hosts that should be excluded from proxying. `NO_PROXY` should contain a comma delimited list of any of the following options.
 
-Option     | Description                                                                                                   | Examples
------------|---------------------------------------------------------------------------------------------------------------|--------------------------------
-IP Address | A single IP address (with optional port)                                                                      | `1.2.3.4` or `1.2.3.4:80`
-CIDR Block | A group of IP addresses that share a network prefix.                                                          | `1.2.3.4/8`
-Domain     | A domain name matches that name and all subdomains. A domain name with a leading "." matches subdomains only. | `example.com` or `.example.com`
-Asterisk   | A single asterisk indicates that no proxying should be done.                                                  | `*`
+| Option     | Description                                                                                                   | Examples                        |
+| ---------- | ------------------------------------------------------------------------------------------------------------- | ------------------------------- |
+| IP Address | A single IP address (with optional port)                                                                      | `1.2.3.4` or `1.2.3.4:80`       |
+| CIDR Block | A group of IP addresses that share a network prefix.                                                          | `1.2.3.4/8`                     |
+| Domain     | A domain name matches that name and all subdomains. A domain name with a leading "." matches subdomains only. | `example.com` or `.example.com` |
+| Asterisk   | A single asterisk indicates that no proxying should be done.                                                  | `*`                             |
 
-## PPROF_MUTEX_PROFILING_PERCENT
+## `PPROF_MUTEX_PROFILING_PERCENT`
 
-You can use the `PPROF_MUTEX_PROFILING_PERCENT` environment variable to define the percentage of mutex profiles to retrieve from the pprof mutex endpoint. If you set this variable to `5`, then 5 percent of the mutexes are sampled. The default value is `0.01`.
+You can use the `PPROF_MUTEX_PROFILING_PERCENT` environment variable to define the percentage of mutex profiles to retrieve from the pprof mutex endpoint.
+If you set this variable to `5`, then 5 percent of the mutexes are sampled.
+The default value is `0`.
```

Comment:
The following is the default behavior in code.
```
// Why 1000 because that is what istio defaults to and that seemed reasonable to start with. This is 00.1% sampling.
runtime.SetMutexProfileFraction(1000)
```

```suggestion
By default, the sampling rate is set to `0.1%`
```